v3/lints/cabf_br/lint_sub_cert_valid_time_longer_than_39_months.go

```diff
@@ -42,7 +42,7 @@ func (l *subCertValidTimeLongerThan39Months) CheckApplies(c *x509.Certificate) b
 }
 
 func (l *subCertValidTimeLongerThan39Months) Execute(c *x509.Certificate) *lint.LintResult {
-	if c.NotBefore.AddDate(0, 39, 0).Before(c.NotAfter) {
+	if util.BeforeOrOn(c.NotBefore.AddDate(0, 39, 0), c.NotAfter) {
```

Comment:
Is this actually correct to make this change?  The Description of the lint says that the lint validity period must be "no greater than 39 months".  A certificate that is valid for exactly 39 months doesn't have a validity period greater than 39 months so I think the original code is the correct version here.